hack/operatorhub/operatorhub-pr-template.md

```diff
@@ -0,0 +1,50 @@
+Thanks submitting your Operator. Please check below list before you create your Pull Request.
```

Comment:
do we need this file?

Reply:
It will be used as a PR description template for the nightly bundle publishing